cltk/stem/middle_english/stem.py

```diff
@@ -4,11 +4,13 @@
 __license__ = 'MIT License. See LICENSE.'
 
 """
-Stemming present a significant challenge in ME, as it is exceptionally difficult to account for the orthographical
- variations sometimes even occurring within a single text. The affix algorithm attempts to account for variations in
- spelling, but still Mostly relies on a relatively narrow hard-coded list (Middle English Dictionary(MED)
- https://quod.lib.umich.edu/m/med/).
-  TODO: Improve on the affix stemmer by implementing an accurate spell checker (Levenshtein Automata?)
+Stemming present a significant challenge in ME, as it is exceptionally
+difficult to account for the orthographical variations sometimes even
+occurring within a single text. The affix algorithm attempts to account
+for variations in spelling, but still Mostly relies on a relatively narrow
+hard-coded list (Middle English Dictionary(MED) https://quod.lib.umich.edu/m/med/)
+
+  TODO: Improve on the affix stemmer by implementing an accurate spell checker
   TODO: Implement a stochastic algorithm/Implement overarching stemmer class
```

Comment:
I like the addition of TODO comments, but maybe they belong outside of a docstring; everything in a docstring will get pulled into generated documentation.
We currently aren't pulling everything into documentation but that will likely change.
On many projects, each TODO item would ideally be placed in Github issues; however, since you are in active development of this content, that seems a bit overkill.
So maybe just add the TODO comments in the future as regular comments.
No need to make them inline documentation comments either ;-)

Wondering what @kylepjohnson and @jtauber think about this.